packages/ui-components/src/components/Map/Legend/Legend.js

```diff
@@ -56,40 +56,48 @@ const getLegendComponent = measureType => {
 
 export const Legend = React.memo(
   ({
-    serieses,
     className,
+    measureInfo: baseMeasureInfo,
     setValueHidden,
     hiddenValues,
-    currentMeasureIds,
-    displayedMeasureIds,
+    currentMapOverlayIds,
+    displayedMapOverlayIds,
+    seriesesKey = 'serieses',
```

Comment:
lesmis changes and uses `serieses` while tupaia uses `measureOptions` from backend response. We can do a clean up in tupaia to get rid of `measureOptions`.

Reply:
Sure: https://linear.app/bes/issue/PHX-120/change-measureoptions-to-serieses